modules/passkey/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@safe-global/safe-passkey",
-  "version": "0.2.1",
+  "version": "0.2.2",
```

Comment:
I think we should use 0.2.1-1 here, since it is an update to supporting libraries and not SC code. Otherwise, it could get confusing WRT `safe-modules-deployments`.

Reply:
Fair enough, will change